### PR TITLE
refactor(checkbox): implement design tokens spec

### PR DESCRIPTION
**Pull request by badejayayesubabu**
## Description

`<hx-checkbox>`: refactor with design tokens

### What are the relevant story cards/tickets? Any additional PRs or other references?

Jira: SURF-2201

## Before you request a review for this PR:

- [x] For UI changes, did you manually test in recent versions of modern browsers (Chrome, Firefox, and Safari)?
- [x] For UI changes, did you manually test in IE11 and legacy Edge?
- [ ] Did you add component tests for any new code?
- [ ] Did you run the component unit tests via `yarn test` to ensure all tests passed?
- [ ] Did you include a screenshot of the component tests?
- [ ] If you changed/added functionality, did you update the demo page and documentation?
- [ ] If needed, did you add or modify the demo test page to test the changed/added functionality?
- [x] Did you assign reviewers?
- [x] In Jira, have you linked to this PR on the ticket(s)?

### CODE REVIEW

File: src/scss/components/checkbox/mixins/_hxCheckboxControl.scss [hunk 1]
```diff
@@ -9,29 +9,31 @@
...
+      background-color: $rax-component-checkbox-hover-background-color;
+      border-color: $rax-component-checkbox-hover-border-color;
       color: $blue-700;
+      color: $rax-component-checkbox-hover-border-color;
     }
   }
   @else {
     color: $gray-500;
-    color: var(--hxCheckbox-color, $gray-500);
+    color: $rax-component-checkbox-default-border-color;
```
Comment (100stacks) [inline src/scss/components/checkbox/mixins/_hxCheckboxControl.scss:29]: Please continue to use the IE11 fallback pattern.  See above.

File: src/scss/components/checkbox/mixins/_hxCheckboxControl.scss [hunk 2]
```diff
@@ -9,29 +9,31 @@
...
 
     &::-ms-check {
       background-color: $gray-0;
-      background-color: var(--hxCheckbox-bgcolor, $gray-0);
+      background-color: $rax-component-checkbox-default-background-color;
```
Comment (100stacks) [inline src/scss/components/checkbox/mixins/_hxCheckboxControl.scss:33]: Please continue to use the IE11 fallback pattern.  See above.

File: src/scss/components/checkbox/mixins/_hxCheckboxControl.scss [hunk 3]
```diff
@@ -9,29 +9,31 @@
...
       border-color: currentColor;
-      border-style: solid;
-      border-width: 1px;
+      border-style: $rax-component-input-base-border-style;
```
Comment (100stacks) [inline src/scss/components/checkbox/mixins/_hxCheckboxControl.scss:35]: Let's not mix component design tokens. We should annotate missing definitions instead.

File: src/scss/components/checkbox/mixins/_hxCheckboxControl.scss [hunk 4]
```diff
@@ -44,7 +46,7 @@
 
     &::-ms-check {
       border-color: $red-status-500;
-      border-color: var(--hxCheckbox-invalid-bordercolor, $red-status-500);
+      border-color: $rax-component-checkbox-invalid-border-color;
```
Comment (100stacks) [inline src/scss/components/checkbox/mixins/_hxCheckboxControl.scss:49]: Please continue to use the IE11 fallback pattern. See above.

File: src/scss/components/checkbox/mixins/_hxCheckboxControl.scss [hunk 5]
```diff
@@ -54,13 +56,15 @@
   }
   @else {
     color: $red-status-500;
-    color: var(--hxCheckbox-invalid-color, $red-status-500);
+    color: $rax-component-checkbox-invalid-border-color;
```
Comment (100stacks) [inline src/scss/components/checkbox/mixins/_hxCheckboxControl.scss:59]: Please continue to use the IE11 fallback pattern. See above.

File: src/scss/components/checkbox/mixins/_hxCheckboxControl.scss [hunk 6]
```diff
@@ -54,13 +56,15 @@
...
     background-color: $red-status-100;
+    background-color: $rax-color-status-critical-100;
```
Comment (100stacks) [inline src/scss/components/checkbox/mixins/_hxCheckboxControl.scss:61]: Please continue to use the IE11 fallback pattern. See above.

File: src/scss/components/checkbox/mixins/_hxCheckboxControl.scss [hunk 7]
```diff
@@ -54,13 +56,15 @@
...
 
     &::-ms-check {
-      border-width: 1px;
-      border-color: $red-status-500 !important;
+      border-width: $rax-component-checkbox-base-border-width;
+      border-color: $rax-component-checkbox-invalid-border-color !important;
```
Comment (100stacks) [inline src/scss/components/checkbox/mixins/_hxCheckboxControl.scss:65]: Please continue to use the IE11 fallback pattern. See above.

File: src/scss/components/checkbox/mixins/_hxCheckboxControl.scss [hunk 8]
```diff
@@ -54,13 +56,15 @@
...
       color: $red-status-500 !important;
+      color: $rax-component-checkbox-invalid-border-color !important;
```
Comment (100stacks) [inline src/scss/components/checkbox/mixins/_hxCheckboxControl.scss:67]: Please continue to use the IE11 fallback pattern. See above.

File: src/scss/components/checkbox/mixins/_hxCheckboxControl.scss [hunk 9]
```diff
@@ -70,25 +74,27 @@
   @if $pseudo-state == focus {
     &::-ms-check {
       border-color: $gray-500;
-      border-color: var(--hxCheckbox-disabled-bordercolor, $gray-500);
+      border-color: $rax-component-checkbox-disabled-border-color;
```
Comment (100stacks) [inline src/scss/components/checkbox/mixins/_hxCheckboxControl.scss:77]: Please continue to use the IE11 fallback pattern. See above.

File: src/scss/components/checkbox/mixins/_hxCheckboxControl.scss [hunk 10]
```diff
@@ -70,25 +74,27 @@
...
       box-shadow: $focus-glow-disabled;
     }
   }
   @else if $pseudo-state == hover {
     &::-ms-check {
       background-color: $gray-0;
-      border-color: $gray-500;
+      border-color: $rax-component-checkbox-disabled-border-color;
```
Comment (100stacks) [inline src/scss/components/checkbox/mixins/_hxCheckboxControl.scss:84]: Please continue to use the IE11 fallback pattern. See above.

File: src/scss/components/checkbox/mixins/_hxCheckboxControl.scss [hunk 11]
```diff
@@ -70,25 +74,27 @@
...
     &::-ms-check {
       border-color: $gray-500;
-      border-color: var(--hxCheckbox-disabled-bordercolor, $gray-500);
+      border-color: $rax-component-checkbox-disabled-border-color;
       box-shadow: $focus-glow-disabled;
     }
   }
   @else if $pseudo-state == hover {
     &::-ms-check {
       background-color: $gray-0;
```
Comment (100stacks) [inline src/scss/components/checkbox/mixins/_hxCheckboxControl.scss:83]: Please update to design token color for $gray-0.

File: src/scss/components/checkbox/mixins/_hxCheckboxControl.scss [hunk 12]
```diff
@@ -70,25 +74,27 @@
...
-      border-color: $gray-500;
+      border-color: $rax-component-checkbox-disabled-border-color;
       color: $gray-500;
+      color: $rax-component-checkbox-disabled-background-color;
```
Comment (100stacks) [inline src/scss/components/checkbox/mixins/_hxCheckboxControl.scss:86]: Please continue to use the IE11 fallback pattern. See above.

File: src/scss/components/checkbox/mixins/_hxCheckboxControl.scss [hunk 13]
```diff
@@ -70,25 +74,27 @@
...
     }
   }
   @else {
     cursor: not-allowed;
 
     &::-ms-check {
       background-color: $gray-0 !important; // simplest way of styling
-      border-color: $gray-500;
-      border-width: 1px;
+      border-color: $rax-component-checkbox-disabled-border-color;
```
Comment (100stacks) [inline src/scss/components/checkbox/mixins/_hxCheckboxControl.scss:94]: Please continue to use the IE11 fallback pattern. See above.

File: src/scss/components/checkbox/mixins/_hxCheckboxControl.scss [hunk 14]
```diff
@@ -70,25 +74,27 @@
...
       color: $gray-500;
+      color: $rax-component-checkbox-disabled-background-color;
     }
   }
   @else {
     cursor: not-allowed;
 
     &::-ms-check {
       background-color: $gray-0 !important; // simplest way of styling
```
Comment (100stacks) [inline src/scss/components/checkbox/mixins/_hxCheckboxControl.scss:93]: Please use design token color value for $gray-0.

File: src/scss/components/checkbox/mixins/_hxCheckboxControl.scss [hunk 15]
```diff
@@ -70,25 +74,27 @@
...
-      border-color: $gray-500;
-      border-width: 1px;
+      border-color: $rax-component-checkbox-disabled-border-color;
+      border-width: $rax-component-checkbox-base-border-width;
       color: $gray-500;
+      color: $rax-component-checkbox-disabled-background-color;
```
Comment (100stacks) [inline src/scss/components/checkbox/mixins/_hxCheckboxControl.scss:97]: Please continue to use the IE11 fallback pattern. See above.

File: src/scss/components/checkbox/mixins/_hxCheckboxFacade.scss [hunk 16]
```diff
@@ -6,51 +6,52 @@
 @mixin __checkboxFacade($pseudo-state: null) {
   @if $pseudo-state == focus {
     border-color: $blue-900;
-    border-color: var(--hxCheckbox-bordercolor-focus, $blue-900);
+    border-color: $rax-color-primary-900;
```
Comment (100stacks) [inline src/scss/components/checkbox/mixins/_hxCheckboxFacade.scss:9]: Please continue to use the IE11 fallback pattern. See above.

File: src/scss/components/checkbox/mixins/_hxCheckboxFacade.scss [hunk 17]
```diff
@@ -6,51 +6,52 @@
...
     box-shadow: $focus-glow;
   }
   @else if $pseudo-state == hover {
-    background-color: var(--hxCheckbox-hover-bgcolor, $gray-0);
-    border-color: var(--hxCheckbox-hover-bordercolor, $blue-700);
-    color: var(--hxCheckbox-hover-color, $blue-700);
+    background-color: $rax-component-checkbox-hover-background-color;
+    border-color: $rax-component-checkbox-hover-border-color;
+    color: $blue-700;
+    color: $rax-component-checkbox-hover-border-color;
```
Comment (100stacks) [inline src/scss/components/checkbox/mixins/_hxCheckboxFacade.scss:16]: Please continue to use the IE11 fallback pattern. See above.

File: src/scss/components/checkbox/mixins/_hxCheckboxFacade.scss [hunk 18]
```diff
@@ -6,51 +6,52 @@
...
   }
 }
 
 // ========== INVALID ==========
 @mixin __checkboxFacade--invalid($pseudo-state: null) {
   @if $pseudo-state == focus {
     border-color: $red-status-500;
-    border-color: var(--hxCheckbox-invalid-bordercolor, $red-status-500);
+    border-color: $rax-component-checkbox-invalid-border-color;
```
Comment (100stacks) [inline src/scss/components/checkbox/mixins/_hxCheckboxFacade.scss:24]: Please continue to use the IE11 fallback pattern. See above.

File: src/scss/components/checkbox/mixins/_hxCheckboxFacade.scss [hunk 19]
```diff
@@ -6,51 +6,52 @@
...
     box-shadow: $focus-glow-invalid;
   }
   @else if $pseudo-state == hover {
     border-color: $red-status-500;
-    border-color: var(--hxCheckbox-invalid-hover-bordercolor, $red-status-500);
+    border-color: $rax-component-checkbox-invalid-border-color;
```
Comment (100stacks) [inline src/scss/components/checkbox/mixins/_hxCheckboxFacade.scss:29]: Please continue to use the IE11 fallback pattern. See above.

File: src/scss/components/checkbox/mixins/_hxCheckboxFacade.scss [hunk 20]
```diff
@@ -6,51 +6,52 @@
...
     color: $red-status-500;
-    color: var(--hxCheckbox-invalid-hover-color, $red-status-500);
+    color: $rax-component-checkbox-invalid-border-color;
```
Comment (100stacks) [inline src/scss/components/checkbox/mixins/_hxCheckboxFacade.scss:31]: Please continue to use the IE11 fallback pattern. See above.

File: src/scss/components/checkbox/mixins/_hxCheckboxFacade.scss [hunk 21]
```diff
@@ -6,51 +6,52 @@
...
   }
   @else {
     border: 1px solid $red-status-500;
-    border: 1px solid var(--hxCheckbox-invalid-border, $red-status-500);
+    border: 1px solid $rax-component-checkbox-invalid-border-color;
```
Comment (100stacks) [inline src/scss/components/checkbox/mixins/_hxCheckboxFacade.scss:35]: Please continue to use the IE11 fallback pattern. See above.

File: src/scss/components/checkbox/mixins/_hxCheckboxFacade.scss [hunk 22]
```diff
@@ -6,51 +6,52 @@
...
     color: $red-status-500;
-    color: var(--hxCheckbox-invalid-color, $red-status-500);
+    color: $rax-component-checkbox-invalid-border-color;
```
Comment (100stacks) [inline src/scss/components/checkbox/mixins/_hxCheckboxFacade.scss:37]: Please continue to use the IE11 fallback pattern. See above.

File: src/scss/components/checkbox/mixins/_hxCheckboxFacade.scss [hunk 23]
```diff
@@ -6,51 +6,52 @@
...
   }
 }
 
 // ========== DISABLED ==========
 @mixin __checkboxFacade--disabled($pseudo-state: null) {
   @if $pseudo-state == focus {
     border-color: $gray-500;
-    border-color: var(--hxCheckbox-disabled-bordercolor, $gray-500);
+    border-color: $rax-component-checkbox-disabled-border-color;
```
Comment (100stacks) [inline src/scss/components/checkbox/mixins/_hxCheckboxFacade.scss:45]: Please continue to use the IE11 fallback pattern. See above.

File: src/scss/components/checkbox/mixins/_hxCheckboxFacade.scss [hunk 24]
```diff
@@ -6,51 +6,52 @@
...
   }
   @else if $pseudo-state == hover {
     background-color: $gray-0;
-    background-color: var(--hxCheckbox-disabled-hover-bgcolor, $gray-0);
+    background-color: $rax-component-checkbox-hover-background-color;
```
Comment (100stacks) [inline src/scss/components/checkbox/mixins/_hxCheckboxFacade.scss:49]: Please continue to use the IE11 fallback pattern. See above.

File: src/scss/components/checkbox/mixins/_hxCheckboxFacade.scss [hunk 25]
```diff
@@ -6,51 +6,52 @@
...
     border-color: $gray-500;
-    border-color: var(--hxCheckbox-disabled-hover-bordercolor, $gray-500);
+    border-color: $rax-component-checkbox-disabled-border-color;
```
Comment (100stacks) [inline src/scss/components/checkbox/mixins/_hxCheckboxFacade.scss:51]: Please continue to use the IE11 fallback pattern. See above.

File: src/scss/components/checkbox/_index.scss [hunk 26]
```diff
@@ -3,8 +3,8 @@
 // ========== BASE ELEMENTS ==========
 hx-checkbox {
   display: inline-block;
-  height: 1rem;
-  width: 1rem;
+  height: $rax-spacing-400;
+  width: $rax-spacing-400;
```
Comment (100stacks) [inline src/scss/components/checkbox/_index.scss:7]: Please add a comment that the design token values are incorrect.

**Review by 100stacks — CHANGES_REQUESTED** [review]

File: src/scss/components/checkbox/_index.scss [hunk 27]
```diff
@@ -27,9 +27,10 @@ hx-checkbox-control {
 
   > label {
     // Checkbox v2.0 specs
-    font-size: 1rem;
+    font-size: $rax-component-checkbox-label-default-font-size;
     color: $gray-950;
-    font-weight: 400; // normal
+    color: $rax-component-label-default-color;
```
[the remark this answers is not in the document]
Comment (badejayayesubabu) [reply src/scss/components/checkbox/_index.scss:31]: Changes have been done, please review.

File: src/scss/components/checkbox/_index.scss [hunk 28]
```diff
@@ -81,11 +82,11 @@ hx-checkbox-control {
 // ========== PRISTINE ==========
 hx-checkbox {
   background-color: $gray-0;
-  background-color: var(--hxCheckbox-bgcolor, $gray-0);
+  background-color: $rax-component-checkbox-default-background-color;
```
[the remark this answers is not in the document]
Comment (badejayayesubabu) [reply src/scss/components/checkbox/_index.scss:84]: Changes have been done, please review.

File: src/scss/components/checkbox/_index.scss [hunk 29]
```diff
@@ -81,11 +82,11 @@ hx-checkbox-control {
...
   border-color: $gray-500;
-  border-color: var(--hxCheckbox-border, $gray-500);
-  border-style: solid;
-  border-width: 1px;
+  border-color: $rax-component-checkbox-default-border-color;
```
[the remark this answers is not in the document]
Comment (badejayayesubabu) [reply src/scss/components/checkbox/_index.scss:86]: Changes have been done, please review.

File: src/scss/components/checkbox/_index.scss (hunk 26 above)
Comment (badejayayesubabu) [reply src/scss/components/checkbox/_index.scss]: Changes have been done, please review.

File: src/scss/components/checkbox/mixins/_hxCheckboxFacade.scss (hunk 25 above)
Comment (badejayayesubabu) [reply src/scss/components/checkbox/mixins/_hxCheckboxFacade.scss:52]: Changes have been done, please review.

File: src/scss/components/checkbox/mixins/_hxCheckboxFacade.scss (hunk 24 above)
Comment (badejayayesubabu) [reply src/scss/components/checkbox/mixins/_hxCheckboxFacade.scss:50]: Changes have been done, please review.

File: src/scss/components/checkbox/mixins/_hxCheckboxFacade.scss (hunk 23 above)
Comment (badejayayesubabu) [reply src/scss/components/checkbox/mixins/_hxCheckboxFacade.scss:46]: Changes have been done, please review.

File: src/scss/components/checkbox/mixins/_hxCheckboxFacade.scss (hunk 22 above)
Comment (badejayayesubabu) [reply src/scss/components/checkbox/mixins/_hxCheckboxFacade.scss]: Changes have been done, please review.

File: src/scss/components/checkbox/_index.scss [hunk 30]
```diff
@@ -81,11 +82,11 @@ hx-checkbox-control {
...
+  border-style: $rax-component-tab-base-border-style;
```
[the remark this answers is not in the document]
Comment (badejayayesubabu) [reply src/scss/components/checkbox/_index.scss]: Changes have been done, please review.

File: src/scss/components/checkbox/_index.scss [hunk 31]
```diff
@@ -129,15 +130,15 @@ hx-checkbox-control {
     &:checked,
     &:indeterminate {
       color: $blue-700;
-      color: var(--hxCheckbox-indeterminate-color, $blue-700);
+      color: $rax-component-checkbox-checked-background-color;// in tokens it is color
 
       ~ label > hx-checkbox {
         background-color: $blue-700;
-        background-color: var(--hxCheckbox-indeterminate-label-bgcolor, $blue-700);
+        background-color: $rax-component-checkbox-checked-background-color;
```
[the remark this answers is not in the document]
Comment (badejayayesubabu) [reply src/scss/components/checkbox/_index.scss:136]: Changes have been done, please review.

File: src/scss/components/checkbox/_index.scss [hunk 32]
```diff
@@ -129,15 +130,15 @@ hx-checkbox-control {
...
         border-color: $blue-700;
-        border-color: var(--hxCheckbox-indeterminate-label-bordercolor, $blue-700);
+        border-color: $rax-component-checkbox-checked-border-color;
```
[the remark this answers is not in the document]
Comment (badejayayesubabu) [reply src/scss/components/checkbox/_index.scss:138]: Changes have been done, please review.

File: src/scss/components/checkbox/_index.scss [hunk 33]
```diff
@@ -129,15 +130,15 @@ hx-checkbox-control {
...
         color: $gray-0;
-        color: var(--hxCheckbox-indeterminate-label-color, $gray-0);
+        color: $rax-component-checkbox-default-background-color;
```
[the remark this answers is not in the document]
Comment (badejayayesubabu) [reply src/scss/components/checkbox/_index.scss]: Changes have been done, please review.

File: src/scss/components/checkbox/_index.scss [hunk 34]
```diff
@@ -178,13 +179,13 @@ hx-checkbox-control.hxInvalid > input[type="checkbox"]:enabled {
   &:checked,
   &:indeterminate {
     color: $red-status-500;
-    color: var(--hxCheckbox-invalid-checked-color, $red-status-500);
+    color: $rax-color-status-critical-500;
```
[the remark this answers is not in the document]
Comment (badejayayesubabu) [reply src/scss/components/checkbox/_index.scss:181]: Changes have been done, please review.

File: src/scss/components/checkbox/mixins/_hxCheckboxControl.scss [hunk 35]
```diff
@@ -9,29 +9,31 @@
...
-    color: var(--hxCheckbox-hover-color, $blue-700);
+    color: $rax-component-checkbox-hover-border-color;
 
     &::-ms-check {
       background-color: $gray-0;
-      border-color: $blue-700;
+      background-color: $rax-component-checkbox-hover-background-color;
+      border-color: $rax-component-checkbox-hover-border-color;
       color: $blue-700;
+      color: $rax-component-checkbox-hover-border-color;
```
[the remark this answers is not in the document]
Comment (badejayayesubabu) [reply src/scss/components/checkbox/mixins/_hxCheckboxControl.scss]: Changes have been done, please review.

File: src/scss/components/checkbox/mixins/_hxCheckboxControl.scss [hunk 36]
```diff
@@ -9,29 +9,31 @@
...
   @else if $pseudo-state == hover {
     color: $blue-700;
-    color: var(--hxCheckbox-hover-color, $blue-700);
+    color: $rax-component-checkbox-hover-border-color;
 
     &::-ms-check {
       background-color: $gray-0;
-      border-color: $blue-700;
+      background-color: $rax-component-checkbox-hover-background-color;
+      border-color: $rax-component-checkbox-hover-border-color;
```
[the remark this answers is not in the document]
Comment (badejayayesubabu) [reply src/scss/components/checkbox/mixins/_hxCheckboxControl.scss:22]: Changes have been done, please review.

File: src/scss/components/checkbox/mixins/_hxCheckboxControl.scss [hunk 37]
```diff
@@ -9,29 +9,31 @@
...
   }
   @else if $pseudo-state == hover {
     color: $blue-700;
-    color: var(--hxCheckbox-hover-color, $blue-700);
+    color: $rax-component-checkbox-hover-border-color;
 
     &::-ms-check {
       background-color: $gray-0;
-      border-color: $blue-700;
+      background-color: $rax-component-checkbox-hover-background-color;
```
[the remark this answers is not in the document]
Comment (badejayayesubabu) [reply src/scss/components/checkbox/mixins/_hxCheckboxControl.scss:20]: Changes have been done, please review.

File: src/scss/components/checkbox/mixins/_hxCheckboxControl.scss [hunk 38]
```diff
@@ -9,29 +9,31 @@
...
     &::-ms-check {
       border-color: $blue-700;
-      border-color: var(--hxCheckbox-focus-bordercolor, $blue-700);
+      border-color: $rax-component-checkbox-focus-border-color;
     }
   }
   @else if $pseudo-state == hover {
     color: $blue-700;
-    color: var(--hxCheckbox-hover-color, $blue-700);
+    color: $rax-component-checkbox-hover-border-color;
```
[the remark this answers is not in the document]
Comment (badejayayesubabu) [reply src/scss/components/checkbox/mixins/_hxCheckboxControl.scss]: Changes have been done, please review.

File: src/scss/components/checkbox/mixins/_hxCheckboxControl.scss [hunk 39]
```diff
@@ -9,29 +9,31 @@
 
     &::-ms-check {
       border-color: $blue-700;
-      border-color: var(--hxCheckbox-focus-bordercolor, $blue-700);
+      border-color: $rax-component-checkbox-focus-border-color;
```
[the remark this answers is not in the document]
Comment (badejayayesubabu) [reply src/scss/components/checkbox/mixins/_hxCheckboxControl.scss:11]: Changes have been done, please review.

File: src/scss/components/checkbox/_index.scss [hunk 40]
```diff
@@ -225,11 +226,11 @@ hx-checkbox-control > input[type="checkbox"]:indeterminate:disabled,
 hx-checkbox-control > input[type="checkbox"]:checked:disabled {
   ~ label > hx-checkbox {
     background-color: $gray-500;
-    background-color: var(--hxCheckbox-disabled-checked-label-bgcolor, $gray-500);
+    background-color: $rax-component-checkbox-disabled-background-color;
     border-color: $gray-500;
-    border-color: var(--hxCheckbox-disabled-checked-label-bordercolor, $gray-500);
+    border-color: $rax-component-checkbox-disabled-border-color;
```
[the remark this answers is not in the document]
Comment (badejayayesubabu) [reply src/scss/components/checkbox/_index.scss:230]: Changes have been done, please review.

File: src/scss/components/checkbox/_index.scss [hunk 41]
```diff
@@ -129,15 +130,15 @@ hx-checkbox-control {
     &:checked,
     &:indeterminate {
       color: $blue-700;
-      color: var(--hxCheckbox-indeterminate-color, $blue-700);
+      color: $rax-component-checkbox-checked-background-color;// in tokens it is color
```
[the remark this answers is not in the document]
Comment (badejayayesubabu) [reply src/scss/components/checkbox/_index.scss]: Changes have been done, please review.

File: src/scss/components/checkbox/_index.scss [hunk 42]
```diff
@@ -178,13 +179,13 @@ hx-checkbox-control.hxInvalid > input[type="checkbox"]:enabled {
   &:checked,
   &:indeterminate {
     color: $red-status-500;
-    color: var(--hxCheckbox-invalid-checked-color, $red-status-500);
+    color: $rax-color-status-critical-500;
 
     ~ label > hx-checkbox {
       background-color: $red-status-100;
-      background-color: var(--hxCheckbox-invalid-checked-bgcolor, $red-status-100);
+      background-color: $rax-color-status-critical-100;
```
[the remark this answers is not in the document]
Comment (badejayayesubabu) [reply src/scss/components/checkbox/_index.scss:185]: Changes have been done, please review.

File: src/scss/components/checkbox/_index.scss [hunk 43]
```diff
@@ -178,13 +179,13 @@ hx-checkbox-control.hxInvalid > input[type="checkbox"]:enabled {
...
       border-color: $red-status-500;
-      border-color: var(--hxCheckbox-invalid-checked-label-bordercolor, $red-status-500);
+      border-color: $rax-component-checkbox-invalid-border-color;
```
[the remark this answers is not in the document]
Comment (badejayayesubabu) [reply src/scss/components/checkbox/_index.scss:187]: Changes have been done, please review.

File: src/scss/components/checkbox/_index.scss [hunk 44]
```diff
@@ -225,11 +226,11 @@ hx-checkbox-control > input[type="checkbox"]:indeterminate:disabled,
 hx-checkbox-control > input[type="checkbox"]:checked:disabled {
   ~ label > hx-checkbox {
     background-color: $gray-500;
-    background-color: var(--hxCheckbox-disabled-checked-label-bgcolor, $gray-500);
+    background-color: $rax-component-checkbox-disabled-background-color;
```
[the remark this answers is not in the document]
Comment (badejayayesubabu) [reply src/scss/components/checkbox/_index.scss:228]: Changes have been done, please review.

**Review by 100stacks — APPROVED** [review]
Dev LGTM

**Pull request merged**